lib/bap_disasm/bap_disasm_rec.ml

```diff
@@ -253,12 +253,27 @@ let next dis s =
 
 let stop_on = [`Valid]
 
+let relocate brancher lift mem insn =
```

Comment:
This function is incorrect, first of all, second, it should exist at all. Brancher is not a relocator. It shouldn't perform relocations. 

So, first of all, why it is incorrect. You take a BIL program, that may have N destinations and an output from a brancher, that may have M destinations, and arbitrary say that all destinations of the BIL program is the first destination from the brancher output. It doesn't sound right to me. So this function should be removed.